test/tools/start-services.sh

```diff
@@ -19,9 +19,10 @@ truffle exec scripts/dumpAddresses.js --network ganache
 popd
 
 # Start Parachain
-docker-compose up -d -- parachain
+docker-compose up -d parachain
 tools/wait-for-it.sh localhost:9944 -- echo "Parachain is up"
 
-# Start Relayer
-docker-compose up -d -- relayer
+sleep 10s
 
+# Start Relayer
+docker-compose up -d relayer
```

Comment:
does it make sense to run tests from `test/test/test.js` here?

Reply:
I not sure about what you mean.
Could you please elaborate?

Reply:
Nevermind, I've missed the run script launching both environment and the tests afterwards